Horion/Module/ModuleManager.cpp

```diff
@@ -103,6 +103,7 @@ void ModuleManager::initModules() {
 		this->moduleList.push_back(std::shared_ptr<IModule>(new VanillaPlus()));
 
 		this->moduleList.push_back(std::shared_ptr<IModule>(new FollowPathModule()));
+		this->moduleList.push_back(std::shared_ptr<IModule>(new Tornado()));
```

Comment:
ModuleManager.cpp always has an error with this line unless you save Tornado.cpp with no changes and build while on that page for some reason.